pkg/kubelet/kuberuntime/helpers.go

```diff
@@ -238,7 +238,7 @@ func annotationProfile(profile, profileRootPath string) string {
 	if strings.HasPrefix(profile, v1.SeccompLocalhostProfileNamePrefix) {
 		name := strings.TrimPrefix(profile, v1.SeccompLocalhostProfileNamePrefix)
 		fname := filepath.Join(profileRootPath, filepath.FromSlash(name))
```

Comment:
I wonder what `profileRootPath` will be on windows. Can it have `c:\` prefix?

Reply:
Hm, we don't support seccomps on Windows, so we should probably skip those tests anyways.